Gopkg.toml

```diff
@@ -106,3 +106,7 @@ required = [
   name = "github.com/jteeuwen/go-bindata"
   # latest, unlikely to change as go-bindata is no longer maintained
   revision = "6025e8de665b31fa74ab1a66f2cddd8c0abf887e"
+
+[[constraint]]
+  name = "gopkg.in/yaml.v2"
```

Comment:
reason?

Reply:
(let's leave a comment?)

Reply:
Added reason (sigs.k8s.io/yaml needs a more updated version, but lacks that definition. I opened https://github.com/kubernetes-sigs/yaml/pull/11 to add that)

Reply:
Thanks!